utils/get_r_u_strings.js

```diff
@@ -0,0 +1,10 @@
+function get_r_u_strings(rankedStatus, speedStatus) {
+    const r_u = rankedStatus ? "ranked" : "unranked";
```

Comment:
Can we not just return `r_u` and let the code on the outside add `for` and `from`? That would also eliminate `speedStatus`.

Reply:
You could then do:
```Js
const noPauseMsg = `Pausing not allowed for ${get_r_u_string(this.state.ranked)} games. Resuming game.`;
```
in other code, instead of the current:
```Js
const r_u_strings = get_r_u_strings(this.state.ranked);
const noPauseMsg = `Pausing not allowed ${r_u_strings.for_r_u_games}. Resuming game.`;
```

Reply:
Oh i see how you use this result in `connection.js`. It makes more sense there and i would keep it, but instead extract the `get_r_u_string` method so you can use just that one in `game.js`.

Reply:
yes, its very very convenient in connection.js
but for game.js we use it only once, so we can maybe directly put the result in msg

theres an intermediate way for readability:
```
const for_r_u_games = get_r_u_strings(this.state.ranked);
const noPauseMsg = `Pausing not allowed ${for_r_u_games}. Resuming game.`;
```

Reply:
```Js
const { for_r_u_games } = get_r_u_strings(this.state.ranked);
```
:D
But i saw you removed the need for this function entirely from the pause code. Nice!

Reply:
yeees i'm so smart thanks to your teachings and your guidance xD